framework/build/enyo-build.js

```diff
@@ -5872,8 +5866,7 @@ return enyo.fetchConfigFile("$enyo/../framework_config.json");
 }, enyo.fetchDeviceInfo = function() {
 return window.PalmSystem ? JSON.parse(PalmSystem.deviceInfo) : undefined;
 }, enyo.requiresWindow(function() {
-if ((window.PalmSystem) && (window.PalmSystem.getResource) && (!window.palmGetResource)) {window.palmGetResource = PalmSystem.getResource;}
```

Comment:
Why remove the check for palmSystem? Wouldn't this break compat on other devices?

Reply:
Seems it's not removed, just moved around, see next line. I got confused here too

Reply:
ah, ok.